modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/utils/constants.ts

```diff
@@ -14,12 +14,16 @@
 
 const IMAGES_PATH = Liferay.ThemeDisplay.getPathThemeImages();
```

Comment:
hey @brianchandotcom this is the only const that is not sorted. I didn't do it because otherwise I would have to rename it and this would affect many files. But all the other consts are sorted